src/Adapter/PlatformServices.Desktop/Utilities/DeploymentUtility.cs

```diff
@@ -296,6 +188,20 @@ private IEnumerable<string> Deploy(IList<DeploymentItem> deploymentItems, string
             return warnings;
         }
 
+        /// <summary>
+        /// Get root deployment directory
+        /// </summary>
+        /// <param name="baseDirectory">The base directory.</param>
+        /// <returns>Root deployment directory.</returns>
+        protected override string GetRootDeploymentDirectory(string baseDirectory)
+        {
+            string dateTimeSufix = DateTime.Now.ToString("yyyy-MM-dd HH:mm:ss", DateTimeFormatInfo.InvariantInfo);
+            string directoryName = string.Format(CultureInfo.CurrentCulture, Resource.TestRunName, DeploymentFolderPrefix, Environment.UserName, dateTimeSufix);
```

Comment:
>string.Format(CultureI [](start = 35, length = 22)

won't this be too long .. or is this something we are doing already..?

Reply:
That's how we've been doing already.